examples/step-1/step-1.cc

```diff
@@ -143,46 +143,61 @@ void second_grid ()
-      // triangulations, cells aren't stored as an array, so simple pointers
-      // do not work, but one can generalize pointers to iterators (see <a
-      // href="http://en.wikipedia.org/wiki/Iterator#C.2B.2B">this wikipedia
-      // link</a> for more information). We will then get an iterator to the
-      // first cell and iterate over all of the cells until we hit the last
-      // one.
+      // Next, we need to loop over the active cells of the triangulation. You
+      // can think of a triangulation as a collection of cells. If it was an
+      // an array, you would just get a pointer that you increment from one
+      // element to the next using the operator `++`. The cells of a
+      // triangulation aren't stored as a simple array, but the concept of an
+      // <i>iterator</i> generalizes how pointers work to arbitrary collections
+      // of objects (see <a href=
+      // "http://en.wikipedia.org/wiki/Iterator#C.2B.2B">wikipedia</a> for more
+      // information). Typically, any container type in C++ will return an
+      // iterator pointing to the start of the collection with a method called
+      // `begin`, and an iterator point to 1 past the end of the collection with
+      // a method called `end`. We can increment an iterator `it` with the
+      // operator `++it`, dereference it to get the underlying data with `*it`,
+      // and check to see if we're done by comparing `it != collection.end()`.
       //
       // The second important piece is that we only need the active cells.
       // Active cells are those that are not further refined, and the only
-      // ones that can be marked for further refinement, obviously. deal.II
-      // provides iterator categories that allow us to iterate over <i>all</i>
-      // cells (including the parent cells of active ones) or only over the
-      // active cells. Because we want the latter, we need to choose
-      // Triangulation::active_cell_iterator as data type.
+      // ones that can be marked for further refinement. deal.II provides
+      // iterator categories that allow us to iterate over <i>all</i> cells
+      // (including the parent cells of active ones) or only over the active
+      // cells. Because we want the latter, we need to call the method
+      // Triangulation::active_cell_iterators().
       //
-      // Finally, by convention, we almost always use the names
-      // <code>cell</code> and <code>endc</code> for the iterator pointing to
-      // the present cell and to the "one-past-the-end" iterator. This is, in
-      // a sense a misnomer, because the object is not really a "cell": it is
-      // an iterator/pointer to a cell. We should really have started to call
-      // these objects <code>cell_iterator</code> when deal.II started in
-      // 1998, but it is what it is.
+      // Putting all of this together, we can loop over all the active cells of
+      // a triangulation with
+      // @code{.cpp}
```

Comment:
I just noticed that with doxygen 1.8.13 the indentation for this code block is lost. Apparently it's a [known issue](https://stackoverflow.com/questions/11190838/doxygen-code-tag-kills-indentation). I haven't found a good workaround. Suggestions?

Reply:
That's a problem we're going to have everywhere. Let's not worry about it as part of the current PR

Reply:
For the record, we currently use 1.8.9.1 for the website.